hack/create_graph_schema.py

```diff
@@ -0,0 +1,46 @@
+import os
+import requests
+import json
+import time
+import logging
+
+logger = logging.getLogger(__name__)
+
+GREMLIN_SERVER_URL_REST = "http://{host}:{port}".format(
+                          host=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_HOST", "localhost"),
+                          port=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_PORT", "8182"))
+
+GRAPH_INITIALIZED = False
+SCHEMA_CREATED = False
+# wait for graph to be initialized
+while not GRAPH_INITIALIZED:
+    try:
+        print("Calling GREMLIN - " + GREMLIN_SERVER_URL_REST)
```

Comment:
please use logging here instead of print

Reply:
Tried with logging, but while tailing the logs, it will print to the console only when it exits. Print looks suitable here. Again, this is just for user's information when they want to tail the logs.

Reply:
Can you please elaborate how users are supposed to take advantage of print here? It acts the same way as print it's just unified. Your problem might be with a) setup of logging here b) level of logging.

Reply:
this code is part of worker and should use same logging configuration as `worker`

Reply:
got clarification from Frido.

Reply:
How would you like to accomplish such things? It's a simple separate process. If you run a web browser besides worker code on your local setup, does it mean that the browser should report errors the same way how worker does?

Reply:
Done.

Reply:
No special treatment needed. Just want to print to console for any user to follow in local deployment, about whether graph_schema_creation is done or not.